openprescribing/matrixstore/tests/decorators.py

```diff
@@ -32,9 +41,17 @@ def setUpClass(inner_cls):
         # Have to wrap this in a staticmethod decorator otherwise Python thinks
         # we're trying to create a new class method
         inner_cls._stop_patching = staticmethod(stop_patching)
+        new_settings = override_settings(
+            CACHES={
+                "default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"}
+            }
+        )
+        new_settings.enable()
+        inner_cls._new_settings = new_settings
```

Comment:
Given that we also have to do this in price_per_unit/test_savings.py, is it worth extracting this more generally?  Or too much faff?

Reply:
I did look into this actually. But it turns out the PPU tests are subtlety different (it's to do with the fact we have two distinct `DataFactory` classes) so it isn't straightforward to do this. A clean up for another day, I think.